packages/react/src/analytics/integrations/AnalyticsApi/AnalyticsApi.js

```diff
@@ -0,0 +1,101 @@
+import { integrations, utils } from '@farfetch/blackout-core/analytics';
+import client from '@farfetch/blackout-core/helpers/client';
+
+export default class AnalyticsAPI extends integrations.Integration {
+  static [utils.CONSENT_CATEGORIES_PROPERTY] =
+    utils.DefaultConsentKeys.MARKETING;
+
+  /**
+   * Creates an instance of Analytics Api integration.
+   *
+   * @param {object} options               - User configured options.
+   * @param {object} loadData              - Analytics's load event data.
+   * @param strippedDownAnalytics - Analytics stripped down instance.
+   */
+  constructor(options, loadData, strippedDownAnalytics) {
+    super(options, loadData, strippedDownAnalytics);
+    this.debugMode = utils.getCookie('analyticsAPIDebug');
+  }
+
+  /**
+   * Creates and returns the instance for the Analytics API integration.
+   *
+   * @param {object} options - Integration options.
+   * @param {object} loadData - Analytics' load event data.
+   * @param {object} analytics - Analytics stripped down instance.
+   * @param strippedAnalytics
```

Comment:
Extra @param can be removed. Also, ensure there's an empty line between the last `@param` and `@returns` and that the last argument is document with the name `strippedAnalytics` instead of `analytics`